packages/astro/README.md

```diff
@@ -93,7 +93,21 @@ Routing happens in `src/pages/*`. Every `.astro` or `.md` file in this folder co
 
 #### 🗂 Static Assets
 
-Static assets should be placed in a `public/` folder in your project. You can place any images, fonts, files, or global CSS in here you need to reference.
+Assets such as images, global CSS, fonts and scripts that you want to add to your site can be placed in either:
+* `src/`: Files in here must be used to be part of the build output. (imported into an Astro component, added as an img src, etc).
```

Comment:
This line is interesting... what are the cases covered by `"etc"`? Just curious if it's possible we're missing some.

Reply:
Images, scripts, and link tags are all for now. I wanted to leave it a little vague so that we can add stuff in the future (like picture sources).

Reply:
@natemoo-re is there anything you want me to change here?

Reply:
Nope, sorry for the delay. LGTM!